sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/multi/MultiServiceEventClient.java

```diff
@@ -491,25 +508,17 @@ public TaskStatusResponse taskStatus(Protos.TaskStatus status) {
      * entry if needed.
      */
     private static OfferResources getEntry(
-            Map<String, Map<Protos.OfferID, OfferResources>> map, String serviceName, Protos.Offer offer) {
-        Map<Protos.OfferID, OfferResources> serviceOffers = map.get(serviceName);
-        if (serviceOffers == null) {
-            serviceOffers = new HashMap<>();
-            map.put(serviceName, serviceOffers);
-        }
-        return getEntry(serviceOffers, offer);
+            Map<String, Map<Protos.OfferID, OfferResources>> map,
+            String serviceName, Protos.Offer offer
```

Comment:
Should break these up into three separate lines -- either all on one line or all on separate lines